### ISSUE TITLE
Add VP9 codec support to JVB 2.0

### ISSUE THREAD

**Issue by erotavlasme** [jitsi/jitsi-videobridge#1133]
Hi,
it would be very important to add VP9 codec support to jitsi-meet as discussed [here](https://community.jitsi.org/t/jitsi-meet-performance-comparison-to-hangouts-teams-and-bigbluebutton/22676). There is already a patch for old JVB 1.0 and a [developer is available](https://github.com/jitsi/libjitsi/pull/432#issuecomment-601427750) to adapt it.
Right now, the VP9 codec support would allow to close the gap between jitsi and its competitors.
Thank you

**Comment by bgrozev** [comment]
We would love to support vp9 in jitsi-meet, and it is on our roadmap. But note that the PR in libjitsi meet is just one small step, and full support will require a more substantial effort.

I notice that jitsi-meet running on Chrome canary uses VP8, while appr.tc uses VP9. I haven't looked further, but this should be an easy fix and contributions (or help in understanding why it happens) are welcome.

**Comment by erotavlas-turbo** [comment]
Any idea about the release date? It would be very useful right now.

**Comment by Windsurfer01** [comment]
VP9 is really important to bring down bandwidth requirements.

**Comment by daynis-olman** [comment]
Do we know when we can expect an alpha release for this?
Really excited this is being implemented! This will be a game changer, and really put JITSI Meet on the map.

**Comment by EwoutH** [comment]
In jitsi-media-transform a draft PR is opened to enable VP9: https://github.com/jitsi/jitsi-media-transform/pull/258

**Comment by pdarcos** [comment]
Also, not sure you all are following this [thread](https://community.jitsi.org/t/flagship-vp9-support-notes-and-discussion/33921/29) but it seems the community can help quite a bit but feels ignored and is losing motivation to help. @JonathanLennox I believe you're working on this. Perhaps you can take a look and give Syonyk some feedback?

Good luck and thanks

**Comment by hex-m** [comment]
With support in Safari on [macOS](https://webkit.org/blog/10929/release-notes-for-safari-technology-preview-110/) and [iOS](https://meterpreter.org/apple-has-natively-supported-google-vp9-codec-technology-in-ios-14/) VP9 seems to work on all relevant browsers. So VP9 could actually be the default codec by now.

**Comment by paulmenzel** [comment]
Isn’t it (unfortunately) already the default since JVB 2.1-358-g3abd64a9-1? At least `about:webrtc` in Mozilla Firefox showed something like this.

So the issue about implementing support could be closed?

Anyway, I strongly advice against using VP9 by default, as older hardware with no VP9 hardware acceleration suppord (and often only H264 and VP8 hardware acceleration support), will have a lot of trouble to decode several VP9 streams.

**Comment by bbaldino** [comment]
VP9 definitely isn't the default codec anywhere.  The VP9 PRs in the bridge/JMT haven't been merged yet.

**Comment by alfredonodo** [comment]
Hi,
any new about this important feature? Now it [should be merged](https://github.com/jitsi/jitsi-media-transform/pull/258).

**Comment by damencho** [comment]
The support for VP9 landed few weeks ago in jvb2.

**Comment by paulmenzel** [comment]
I could have sworn VP9 was used already with the unstable packages (`deb https://download.jitsi.org unstable/`) before submitting that merge/pull request. At least that is what I gathered from the browser developer tools and the system load.

Besides that, in my experience, I still prefer VP8 (or even H.264), so old devices can use GPU hardware acceleration for decoding (and even encoding).

(Looks like I already commented that in October 2020. No idea, what I did wrong in my testing.)
